teuthology/task/selinux.py

```diff
@@ -20,6 +20,15 @@ class SELinux(Task):
     must first be enabled and the filesystem must have been labeled.
 
     On teardown, also checks the audit log for any denials.
+    By default the test will fail for any denials seen in audit.log, for the
```

Comment:
This is actually not quite true. Since you're providing a default whitelist (which appears to be mandatory), it won't fail for *any* by default.